### PR TITLE
Tien

### PR DESCRIPTION
**Pull request by TimTran115**
Please double check so I don't mess things up

### CODE REVIEW

**Review by realCalvin — CHANGES_REQUESTED** [review]
Rebase from master. You rebased from Tiffany's branch.
Also, squash commits, https://github.com/wprig/wprig/wiki/How-to-squash-commits

File: blits.sql [hunk 1]
```diff
@@ -65,6 +81,14 @@ primary key(game_no)
...
 -- foreign key(game_no) REFERENCES plays(game_id)
 -- on update cascade
+=======
+type varchar (9)
+	check(type in ('Blackjack', 'Coin Flip', 'Slots')),
+winner varchar(10) not null,
+primary key(game_no),
+-- foreign key(winner) REFERENCES player(player_id)
+UNIQUE(game_no)
+>>>>>>> 624c6daa8b46cedcda8a1f373cbf5831c8725e49
```
Comment (tiffanynn) [inline blits.sql]: these need to be gone

File: blits.sql [hunk 2]
```diff
@@ -65,6 +81,14 @@ primary key(game_no)
 -- on update cascade,
 -- foreign key(game_no) REFERENCES plays(game_id)
 -- on update cascade
+=======
```
Comment (tiffanynn) [inline blits.sql]: same with these

File: blits.sql [hunk 3]
```diff
@@ -52,11 +58,21 @@ primary key(player_id, game_id)
...
 
+<<<<<<< HEAD
+=======
+create table players_list(
+game varchar(9),
+max_capacity int not null
+-- foreign key(game) REFERENCES game(type)
+); -- i don't think this is complete
+
+>>>>>>> 624c6daa8b46cedcda8a1f373cbf5831c8725e49
```
Comment (tiffanynn) [inline blits.sql]: this needs to be gone

File: blits.sql [hunk 4]
```diff
@@ -52,11 +58,21 @@ primary key(player_id, game_id)
 ); 
 -- relationship between new_user and game
 
+<<<<<<< HEAD
+=======
```
Comment (tiffanynn) [inline blits.sql]: same w/ here

File: blits.sql [hunk 5]
```diff
@@ -52,11 +58,21 @@ primary key(player_id, game_id)
 ); 
 -- relationship between new_user and game
 
+<<<<<<< HEAD
```
Comment (tiffanynn) [inline blits.sql]: remove

File: blits.sql [hunk 6]
```diff
@@ -52,11 +58,21 @@ primary key(player_id, game_id)
...
+-- foreign key(game) REFERENCES game(type)
+); -- i don't think this is complete
+
+>>>>>>> 624c6daa8b46cedcda8a1f373cbf5831c8725e49
 create table game(
 game_no varchar(10) not null,
 game_status varchar(10)
 	check (game_status in ('Open', 'In Progress', 'Finished')),
 wager_amt int not null,
+<<<<<<< HEAD
```
Comment (tiffanynn) [inline blits.sql]: remove

**Review by tiffanynn — COMMENTED** [review]
just rebase with the master branch and go through the commits first. lines i commented on need to be removed. it took me awhile to get but it'll work out at the end.

File: blits.sql (hunk 3 above)
Comment (tiffanynn) [reply blits.sql]: only line 69

**Comment by TimTran115** [comment]
Capture
Update profile

**Comment by TimTran115** [comment]
I think I updated everything now

**Review by realCalvin — DISMISSED** [review]
Some changes:

- Can you make it so this page looks like the other ones? (look at the css/colors used)
- Remove status because that's additional work to do & store
- For game history matches can you put like dummy data, so we can see what it'd look like? (maybe try making it so that losses are displayed as red and wins as green somehow)

**Review by hsorathia — APPROVED** [review]

**Pull request merged**